trin-history/src/storage.rs

```diff
@@ -616,7 +615,7 @@ pub mod test {
         }
 
         let bytes = storage.get_total_storage_usage_in_bytes_from_network()?;
-        assert_eq!(1600000, bytes); // 32kb * 50
+        assert_eq!(1603200, bytes); // 32kb * 50 + 64 * 50
```

Comment:
`32kb * 50 + CONTENT_ID_AND_KEY_LENGTH * 50` or `(32kb + CONTENT_ID_AND_KEY_LENGTH) * 50` ... and below on `787`